pkg/controller/common/controller_context.go

```diff
@@ -91,6 +94,7 @@ func CreateControllerContext(cb *clients.Builder, stop <-chan struct{}, targetNa
 		apiextinformers.WithNamespace(targetNamespace), apiextinformers.WithTweakListOptions(assignFilterLabels))
 	configSharedInformer := configinformers.NewSharedInformerFactory(configClient, resyncPeriod()())
 	operatorSharedInformer := operatorinformers.NewSharedInformerFactory(operatorClient, resyncPeriod()())
+	imageSharedInformer := imageinformers.NewSharedInformerFactory(imageClient, resyncPeriod()())
```

Comment:
Would like to think about whether we should filter this to our namespace, there's probably a ton of image traffic.

Reply:
That's a great example of a potential risk!  Let's tighten that up indeed.

Reply:
The number of watches and any unusual increases are being monitored so this is a good callout :+1: